ow_dynamic_terrain/include/memory_ext.h

```diff
@@ -0,0 +1,21 @@
+#ifndef MEMORY_EXT_H
+#define MEMORY_EXT_H
+
+// TODO: consider moving the file to a common place
+
+#include <memory>
+
+#if __cplusplus < 201402L  // if less than C++14 is detected add the extension
+
+namespace std
+{
+template <typename T, typename... Args>
```

Comment:
Please comment on what these parts do.

Reply:
addressed

Reply:
C++11 forgot to add a make_unique function similar to make_shared so I had it added for C++11 only compiles

I think it is beneficial to add this in a common place such that  other modules can make use of it